scripts/unity/header_prepare.py

```diff
@@ -39,9 +39,9 @@ def header_prepare(in_file, out_file, out_wrap_file):
     content = syscall_pattern.sub(r"", content)
 
     syscall_decl_pattern = re.compile(
-        r'__syscall\s+(?:\w+[*\s]+)+(.+?;)',
+        r'__syscall\s+((?:\w+[*\s]+)+(.+?;))',
         re.M | re.S)
-    content = syscall_decl_pattern.sub("", content)
+    content = syscall_decl_pattern.sub(r"\1", content)
```

Comment:
Just strip out `__syscall`, and treat these functions as any other instead?

Reply:
done